run_portal.ps1

```diff
@@ -0,0 +1,27 @@
+# Change to uid2-admin directory, run docker compose and then maven compile
+Set-Location -Path "../uid2-admin" -ErrorAction Stop
+docker compose up -d
+if ($LASTEXITCODE -ne 0) {
+    Write-Error "Docker compose failed in uid2-admin"
+    exit 1
+}
+
+Start-Process "mvn" -ArgumentList "clean compile exec:java", "-Dvertx-config-path=conf/local-config.json", "-Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory", "-Dlogback.configurationFile=conf/logback.xml" -WindowStyle Hidden
```

Comment:
`Start-Process` runs the command in a separate terminal, allow the original terminal to continue exiting. I've decided to hide the separate terminal using `-WindowStyle Hidden` because it doesn't really add much value and is yet another terminal.